packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/testHeadingFollowedByParagraphWithInlineEmphasis.st

```diff
@@ -4,4 +4,8 @@ testHeadingFollowedByParagraphWithInlineEmphasis
 	self
 		assert: '# Title **bold**', Character cr, 'Line *italic*'
 		parsesToRuns: {8. 8. 1. 5. 8}
-		values: {{TextFontReference toFont: (TextStyle default copy fontOfSize: 26). TextEmphasis underlined}. {TextFontReference toFont: (TextStyle default copy fontOfSize: 26). TextEmphasis underlined. TextEmphasis bold.}. {TextFontReference toFont: (TextStyle default copy fontOfSize: 26). TextEmphasis underlined}. {}. {TextEmphasis italic}}
\ No newline at end of file
+		values: {{self headingFontSized: 34}. 
```

Comment:
Indentation discussion? Weird: it looks different on GitHub in comparison to Squeak. Maybe one more tab before the lines below to align it?

Reply:
It looks perfect aligned in my image?